### ISSUE TITLE
Plugin config type incompatible with ESLint `ExtendsElement` type

### ISSUE THREAD

**Issue by jhoermann** [schoero/eslint-plugin-readable-tailwind#306]
### Description

The config type is incompatible with ESLint:
<img width="1416" height="286" alt="Image" src="https://github.com/user-attachments/assets/9648df5d-9d6a-4943-8751-4f6033013038" />

Namely the plugins property:
`plugins: string[];` but expected is `plugins: Record<string, Plugin>;`

### Flavor

Vue

### Code Input

```tsx
`extends: [eslintPluginBetterTailwindcss.configs.recommended],`
```

### Expected Behavior

```tsx
No TS error
```

### Actual Behavior

```tsx
TS error 2322
```

### ESLint Log

```shell
No error in eslint log only TS.
```

### ESLint Config

```typescript
`extends: [eslintPluginBetterTailwindcss.configs.recommended],`
```

### Versions

- ESLint: 9.39.2
- Parser: 10.2.0 (vue)
- Plugin: 4.0.1
- Node: 24.12.0
